### PR TITLE
Rework Context Menu QML to Be Suitable for Tablet

### PR DESCRIPTION
**Pull request by fayeli**
This PR contains a few changes in QML I made to make context menu look and feel like it belongs to the tablet. Things I changed include:

- Addition of a Navigation Bar. It contains a hamburger icon that bring user back to top level menu, and a breadcrumb text indicating which level of the menu user is in.
- Width and Height change to fit in Tablet sreen
- Minor margin layout changes

Test Plan:
1. Download and install Interface from this PR
2. Bring out Tablet, and click the "Menu" button.
3. Confirm that you see the context menu fitting in the Tablet screen, and that there is a top nav bar that says "Menu"
4. Try the different menu options. Confirm each time you click a next level pop up  (such as **Avatar >**), you see the pop up taking over the whole screen. Also confirm that the breadcrumb text update.
5. Try using the navigation elements on the nav bar. The hamburger icon takes you back to top level, and the text take you to the parent level of current level.
5. Repeat and check it works well with both desktop mouse click and stylus in HMD.

hifi-snap-by-faye-on-2017-01-13_09-27-30

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 834e3af879866a081d909ca1f017e39b7a527683 available at: https://gist.github.com/0d851266a7d12b28c9fb8ecfe34a39d3<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9377-834e3af.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9377-834e3af.dmg

**Comment by Nex-Pro** [comment]
I'd like to test this when it is QA ready.

**Comment by sethalves** [comment]
It's not ready for a pass/fail (we know it will fail), but feel free to try it and give us feedback.

**Comment by Nex-Pro** [comment]
Okies

**Comment by Nex-Pro** [comment]
@sethalves  Requesting permission to rant.

**Comment by Nex-Pro** [comment]
I'm not going to comment on the desktop experience of this other than to say  Don't do it ! Also this highlights again just how bad text rendering is on these entities. 

HMD:
Would this work better using the trackpads/joysticks as input devices ? Then there's no need to have Gigantor's icons plastered over a humungous tablet model. 

This whole tablet thing feels kindergarten clunky and mixing the lasers with the stylus makes no sense. 
I'm not a human interface designer but please find some other way to do this, maybe with pie menus centered and locked to the HMD view instead of this awful mess.

Basically take the whole QT project and throw it in the trash can where it  belongs.
